src/msonedrive/Module.md

```diff
@@ -0,0 +1,17 @@
+Connects to Microsoft OneDrive and provides information on the files which have been stored on OneDrive.
+
+# Module Overview
+This module contains operations for accessing the items stored in OneDrive.
+
+## Compatibility
+|                     |    Version     |
+|:-------------------:|:--------------:|
+| Ballerina Language  | 1.2.0   |
+| Microsoftgraph REST API | v1.0          |
+
+## Supported Operations
+- Get item from root directory
+- Get item from a non-root directory
+
+## Sample Application
+Please refer the sample application on the README.
```

Comment:
Shall we add some code snippets of supported API calls? with a small description. like in. This module description appears in central. 
1. https://github.com/ballerina-platform/module-slack/blob/master/src/slack/Module.md
2. https://github.com/ballerina-platform/module-googlespreadsheet/blob/master/src/googleapis.sheets4/Module.md

Reply:
Ok, Done